views/doctorform.ejs

```diff
@@ -29,30 +37,26 @@
     </div>
     </div>
 
-
     <div class="row form-group input_group">
 
       <div class="col-sm-12">
        <input id="specialization" type="specialization" name="specialization" class="form-control" placeholder="specialization">
     </div>
     </div>
 
+     <div class="row form-group input_group">
```

Comment:
The edits to these two divs did nothing but mess up the spacing and add unnecessary blank lines.